software/tests/gateware/test_i2c.py

```diff
@@ -1,6 +1,7 @@
 import unittest
-from amaranth import *
+from amaranth import Elaboratable, Module, Signal
```

Comment:
Our style prefers the use of a glob import for Amaranth specifically:

```suggestion
from amaranth import *
```